src/Onboarding/Setup/PageView.php

```diff
@@ -48,6 +48,24 @@ public function render_template( $template, $data = [] ) {
 		return $output;
 	}
 
+	/**
+	 * @return bool
+	 *
+	 * @since 2.8.0
+	 */
+	public function isStripeSetup() {
+		return \Give\Helpers\Gateways\Stripe::isAccountConfigured();
+	}
+
+	/**
+	 * @return bool
+	 *
+	 * @since 2.8.0
+	 */
+	public function isPayPalSetup() {
+		return false;
```

Comment:
This will be implemented when PayPal is available, otherwise we can add a check for PayPal Standard as a fallback.